### PR TITLE
Load pretrained VGG model for face detection.

### PR DESCRIPTION
**Pull request by baiyfbupt**
Fix https://github.com/PaddlePaddle/models/issues/926

### CODE REVIEW

File: fluid/face_detction/train.py [hunk 1]
```diff
@@ -37,9 +41,17 @@ def train(args,
     place = fluid.CUDAPlace(0) if args.use_gpu else fluid.CPUPlace()
     exe = fluid.Executor(place)
     exe.run(fluid.default_startup_program())
-    #print(fluid.default_main_program())
-    #print(test_program)
-    #fluid.io.save_persistables(exe, "model")
+
+    # load_model.load_and_set_vars(place)
```
Comment (qingqing01) [inline fluid/face_detction/train.py]: The load_mode.py and `vgg/names.map` can be removed, if the model has been saved as Fluid format.

Comment (baiyfbupt) [reply fluid/face_detction/train.py]: Done

File: fluid/face_detction/train.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
+import os
 import numpy as np
 import argparse
 import functools
 
+import load_model as load_model
```
Comment (qingqing01) [inline fluid/face_detction/train.py]: Remove this line.

File: fluid/face_detction/train.py (hunk 1 above)
Comment (qingqing01) [reply fluid/face_detction/train.py]: Also need to remove this line.

File: fluid/face_detction/train.py (hunk 2 above)
Comment (baiyfbupt) [reply fluid/face_detction/train.py]: Done

**Review by qingqing01 — APPROVED** [review]

File: fluid/face_detction/train.py (hunk 1 above)
Comment (baiyfbupt) [reply fluid/face_detction/train.py]: Done

**Pull request merged**